tests/test_metrics/test_pck_accuracy.py

```diff
@@ -1,206 +1,215 @@
-        target = {
-            'Head': 100.0,
-            'Shoulder': 100.0,
-            'Elbow': 100.0,
-            'Wrist': 100.0,
-            'Hip': 100.0,
-            'Knee': 100.0,
-            'Ankle': 100.0,
-            'PCKh': 100.0,
-            'PCKh@0.1': 100.0,
-        }
-        self.assertDictEqual(pckh_results, target)
-
-
-class TestJhmdbPCKAccuracy(TestCase):
-
-    def setUp(self):
-        """Setup some variables which are used in every test method.
-
-        TestCase calls functions in this order: setUp() -> testMethod() ->
-        tearDown() -> cleanUp()
-        """
-        self.batch_size = 8
-        num_keypoints = 15
-        self.predictions = []
-        self.groundtruths = []
-
-        for i in range(self.batch_size):
-            keypoints = np.zeros((1, num_keypoints, 2))
-            keypoints[0, i] = [0.5 * i, 0.5 * i]
-            keypoints_visible = np.ones((1, num_keypoints)).astype(bool)
-            keypoints_visible[0, (2 * i) % 8] = False
-            bbox_size = np.random.random((1, 2)) * 20 * i
-            head_size = np.random.random((1, 2)) * 10 * i
-            torso_size = np.random.random((1, 2)) * 15 * i
-
-            prediction = {'coords': keypoints}
-            groundtruth = {
-                'coords': keypoints,
-                'mask': keypoints_visible,
-                'bbox_size': bbox_size,
-                'head_size': head_size,
-                'torso_size': torso_size
-            }
-
-            self.predictions.append(prediction)
-            self.groundtruths.append(groundtruth)
-
-    def test_init(self):
-        """test metric init method."""
-        # test invalid normalized_items
-        with self.assertRaisesRegex(
-                KeyError, "Should be one of 'bbox', 'head', 'torso'"):
-            JhmdbPCKAccuracy(norm_item='invalid')
-
-    def test_evaluate(self):
-        """test PCK accuracy evaluation metric."""
-        # test normalized by 'bbox_size'
-        jhmdb_pck_metric = JhmdbPCKAccuracy(thr=0.5, norm_item='bbox')
-        pckh_results = jhmdb_pck_metric(self.predictions, self.groundtruths)
-        target = {
-            'PCK@0.5': 1.0,
-            'Head PCK': 1.0,
-            'Sho PCK': 1.0,
-            'Elb PCK': 1.0,
-            'Wri PCK': 1.0,
-            'Hip PCK': 1.0,
-            'Knee PCK': 1.0,
-            'Ank PCK': 1.0,
-            'Mean PCK': 1.0,
-        }
-        self.assertDictEqual(pckh_results, target)
-
-        # test normalized by 'torso_size'
-        jhmdb_tpck_metric = JhmdbPCKAccuracy(thr=0.2, norm_item='torso')
-        tpck_results = jhmdb_tpck_metric(self.predictions, self.groundtruths)
-        target = {
-            'Head tPCK': 1.0,
-            'Sho tPCK': 1.0,
-            'Elb tPCK': 1.0,
-            'Wri tPCK': 1.0,
-            'Hip tPCK': 1.0,
-            'Knee tPCK': 1.0,
-            'Ank tPCK': 1.0,
-            'Mean tPCK': 1.0,
-        }
```

Comment:
It seems like this ut was missing.

Reply:
The `JhmdbPCKAccuracy` is tested in:
![image](https://user-images.githubusercontent.com/32220263/212225982-321fc329-b1db-4231-a2c1-b160e002c00c.png)

Reply:
Updated